interfaces/builtin/i2c.go

```diff
@@ -17,8 +17,23 @@
  *
  */
 
-package boot
+package builtin
 
-var (
-	ImportAssertionsFromSeed = importAssertionsFromSeed
+import (
+	"github.com/snapcore/snapd/interfaces"
 )
+
+const i2cConnectedPlugAppArmor = `
+/dev/i2c-[0-9]* rw,
+`
```

Comment:
@jdstrand Discussed this with @niemeyer and we agreed that we will implement this the same way as the serial-port/hidraw interfaces. @bergotorino is working on that now.